src/lsptoolshost/server/roslynLanguageClient.ts

```diff
@@ -3,13 +3,9 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import {
-    CancellationToken,
-    LanguageClient,
-    LanguageClientOptions,
-    MessageSignature,
-    ServerOptions,
-} from 'vscode-languageclient/node';
+import { CancellationToken, MessageSignature } from 'vscode-jsonrpc';
```

Comment:
Lots of places where types are only being exported from their implementation package.